app/controlplane/api/controlplane/v1/attestation_state.proto

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package controlplane.v1;
+
+option go_package = "github.com/chainloop-dev/chainloop/app/controlplane/api/controlplane/v1;v1";
+
+import "validate/validate.proto";
+
+// This service is used by the CLI to generate attestation
+service AttestationStateService {
```

Comment:
This API endpoint basically implements 

https://github.com/chainloop-dev/chainloop/blob/bec4b2a73a2abd529f8c66f8a26dd8ce94973fb9/internal/attestation/crafter/crafter.go#L43-L54